### PR TITLE
Use indeterminate-linear-indicator

### PR DESCRIPTION
**Pull request by christianmemije**
Depends on https://github.com/learningequality/kolibri/pull/3473
Is this the right branch?

### CODE REVIEW

**Comment by indirectlylit** [comment]
target branch should be develop

File: kolibri_exercise_perseus_plugin/assets/src/widgetSolver/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import logger from 'kolibri.lib.logging';
```
Comment (rtibbles) [inline kolibri_exercise_perseus_plugin/assets/src/widgetSolver/index.js:1]: Is your IDE very opinionated about import order? Should we add a linting rule?

**Review by rtibbles — APPROVED** [review]

File: kolibri_exercise_perseus_plugin/assets/src/widgetSolver/index.js (hunk 1 above)
Comment (christianmemije) [reply kolibri_exercise_perseus_plugin/assets/src/widgetSolver/index.js:1]: Already added haha https://github.com/learningequality/kolibri/pull/3471

Comment (rtibbles) [reply kolibri_exercise_perseus_plugin/assets/src/widgetSolver/index.js:1]: Ah, I missed that - you must love that blame!

**Pull request merged**